images/linux/scripts/installers/chromium.sh

```diff
@@ -1,11 +1,35 @@
 #!/bin/bash -e
 
-# Get URL to download the latest Chromium
-LASTCHANGE_URL="https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Linux_x64%2FLAST_CHANGE?alt=media"
-REVISION=$(curl -s -S $LASTCHANGE_URL)
-ZIP_URL="https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Linux_x64%2F$REVISION%2Fchrome-linux.zip?alt=media"
+source $HELPER_SCRIPTS/install.sh
+
+function GetChromiumRevision {
+    CHROME_VERSION=$1
+
+    URL="https://omahaproxy.appspot.com/deps.json?version=${CHROME_VERSION}"
+    REVISION=$(curl -s $URL | jq -r '.chromium_base_position')
+    # Take the first part of revision variable for search
+    FIRST_PART_OF_REVISION=${revision:0:${#revision}/2}
+    URL="https://www.googleapis.com/storage/v1/b/chromium-browser-snapshots/o?delimiter=/&prefix=Linux_x64/${FIRST_PART_OF_REVISION}"
+    VERSIONS=$(curl -s $URL | jq -r '.prefixes[]' | cut -d "/" -f 2 | sort --version-sort)
+
+    RIGHT_REVISION=$(echo $VERSIONS | cut -f 1 -d " ")
+    for version in $VERSIONS; do
+        if [ $revision -lt  $version ]; then
+            echo $RIGHT_REVISION
+            return
+        fi
+        RIGHT_REVISION=$version
+    done
+    echo $RIGHT_REVISION
+}
+
+CHROME_VERSION=$(google-chrome --version | cut -d " " -f 3)
```

Comment:
I would prefer to get rid of this dependency and get the latest chrome version from the internet

Reply:
Changed it to get the version from `http://omahaproxy.appspot.com/`

Reply:
I think we should install Chromium based on installed Chrome version because we install Chrome Driver that is available with Chrome. So Chromium should be the same version to be compatible with Chrome Driver

Reply:
Why?
We install Chrome from apt. Then we find compatible version of ChromeDriver based on installed Chrome. Chromium should be the same version to be compatible.

If we retrieve Chrome version from the internet - we can catch the case where Chromium and ChromeDriver are not compatible.
Like apt still contains Chrome 89.x and we install ChromeDriver 89.x but `http://omahaproxy.appspot.com/` or other remote site already contains Chrome 90.x that is not uploaded to apt yet. In this case, we will install Chromium 90.x that is not compatible with ChromeDriver.

It is better to rely on Chrome version that is already installed on image because we 100% sure that ChromeDriver is compatible with it.

Reply:
@maxim-lobanov we install chrome from the deb package now https://github.com/actions/virtual-environments/pull/3036
If we rely on chrome then we depend on the other provisioning script and we already have a few complaints about such unclear dependencies

Reply:
Yep, and I am not sure we can retrieve exact version from the link https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
Retrieving version from any other resource won't work for us because they can have different versions

In my opinion, it is better to depend on other provisioning script than having Chromium that is not compatible with ChromeDriver.

As an option, we can install chromium in the same script like Chrome and ChromeDriver but I am not sure that it will be better.

Reply:
Hmm, I like the idea to move chromium installation to the chrome provisioning script. In that case, everything will be in a single script